### PR TITLE
Redirect to edit user after creation

### PR DESCRIPTION
**Pull request by mohsen-alizadeh**
#8589

### CODE REVIEW

**Comment by bbonislawski** [comment]
Thanks for your contribution @mohsen-alizadeh but lots of spec failed, can you try to fix them or find the reason why they fail? https://circleci.com/gh/spree/spree/6706#tests/containers/1

**Comment by artplan1** [comment]
@bbonislawski 
same errors exists in master branch, latest merge commit build failed:
https://circleci.com/gh/spree/spree/6699#tests/containers/1

**Comment by bbonislawski** [comment]
Thanks @artplan1 for noticing. I didn't 😱 Going to fix them asap

**Comment by mohsen-alizadeh** [comment]
@bbonislawski please merge this PR, I'm going to investigating on errors exist on master and I will create another PR for that

**Comment by artplan1** [comment]
also @mohsen-alizadeh can you fix this issue for `user update` too? it should redirect

**Comment by mohsen-alizadeh** [comment]
@artplan1 Done

**Comment by artplan1** [comment]
@mohsen-alizadeh 
thank you.

Did you check it in admin panel locally? I think because of this way (redirect) `flash.now` methods will not work and user will not see any notifications. It should be just `flash` if it's before redirect.

in case of `update` I'm not sure that user will see validation errors on save (if something is wrong)  because of redirect instead of render.

it can be tested in feature specs too

**Comment by bbonislawski** [comment]
Sorry for the problems with specs. I've fixed it in #8604 . Please rebase against current master and push again

**Comment by mohsen-alizadeh** [comment]
@bbonislawski It's ready to merge

File: backend/app/controllers/spree/admin/users_controller.rb [hunk 1]
```diff
@@ -13,7 +13,7 @@ def create
         @user = Spree.user_class.new(user_params)
         if @user.save
           flash.now[:success] = flash_message_for(@user, :successfully_created)
```
Comment (artplan1) [inline backend/app/controllers/spree/admin/users_controller.rb]: `flash.now` will not work here

**Review by artplan1 — CHANGES_REQUESTED** [review]

**Comment by bbonislawski** [comment]
@mohsen-alizadeh please squash the commits and I'll merge it. Thanks 👍

**Comment by artplan1** [comment]
Great job @mohsen-alizadeh 
Thank you

**Comment by bbonislawski** [comment]
tenor-2

**Comment by bbonislawski** [comment]
Thanks @mohsen-alizadeh , you're awesome!

**Pull request merged**